src/index.js

```diff
@@ -13,14 +13,35 @@ export function doubleXTimes ( x, num ) {
   return result;
 }
 
-export function doubleEach ( arr ) {
-  let result = arr.map( double );
+export const doubleEach = x => x.map(double);
 
-  // const result = arr.map( x => double( x ) );
-  // const result = arr.map( function ( x ) {
-  //   return double( x );
-  // });
+export function Animal() {
+    this.clorophyll = false;
+}
+Animal.SOUND = 'generic sound';
+Animal.prototype.speak = function () {return this.constructor.SOUND};
 
-  return result;
+export function Reptile() {
+    this.scales = true;
+}
+
+Reptile.SOUND = 'Roar';
+Reptile.prototype = new Animal();
+Reptile.prototype.constructor = Reptile;
+
+export function Primate() {
+    this.thumbs = true;
 }
+Primate.SOUND = 'Ooh ooh eee!';
+Primate.prototype = new Animal();
```

Comment:
You're looking for `Object.create` here. Much cleaner. Otherwise, `chlorophyll` is defined on the prototype instead of the instance, which isn't in line with what you did with animal. Also see comment above.